as/itests/org.jboss.tools.as.itests/src/org/jboss/tools/as/itests/EJB3SupportVerifierTest.java

```diff
@@ -71,6 +71,7 @@
 		expected.put(IJBossToolingConstants.SERVER_EAP_61, true);
 		expected.put(IJBossToolingConstants.SERVER_EAP_70, true);
 		expected.put(IJBossToolingConstants.SERVER_EAP_71, true);
+		expected.put(IJBossToolingConstants.SERVER_WILDFLY_150, true);
```

Comment:
Out of order. Put WF15 below 14?

Reply:
That was a side effect of your script :)